linode/manager
# M3-5614: Update stories for Tags

## Description

Cleanup the stories for Tags and convert the components to FC

## How to test

- Run `yarn storybook` and go to http://localhost:6006/?path=/docs/ui-elements-tags-tag--tag
- Check the app for regressions
